src/consumer/offsetManager/isInvalidOffset.js

```diff
@@ -1,4 +1,3 @@
 const Long = require('long')
-const isNumber = number => /^-?\d+$/.test(number)
 
-module.exports = offset => !isNumber(offset) || Long.fromValue(offset).compare(0) === -1
+module.exports = offset => !offset || Long.fromValue(offset).isNegative()
```

Comment:
I think in this case 0 will return 'true' (i think this is the reason for failing test)

Reply:
Good point. I have to start trusting the tests a bit more, they still are quite flaky and I didn't notice the failure.